src/plugins/data_views/server/rest_api_routes/internal/fields_for.ts

```diff
@@ -188,5 +188,8 @@ export const registerFieldForWildcard = (
   router.versioned.post({ path, access }).addVersion({ version, validate }, handler);
   router.versioned
     .get({ path, access })
-    .addVersion({ version, validate: { request: { query: querySchema } } }, handler);
+    .addVersion(
+      { version, validate: { request: { query: querySchema }, response: validate.response } },
```

Comment:
@davismcphee - This is why I was so confused. You managed to find a case where there were field conflicts AND code was using a POST method - GET didn't have any response validation which had me really confused when I tried to reproduce the problem.